saleor/graphql/account/types.py

```diff
@@ -41,20 +42,28 @@ def resolve_country(self, info):
 
 
 class User(CountableDjangoObjectType):
-    permissions = graphene.List(
-        PermissionDisplay, description='List of user\'s permissions.')
     addresses = gql_optimizer.field(
-        graphene.List(
-            Address, description='List of all user\'s addresses.'),
+        graphene.List(Address, description='List of all user\'s addresses.'),
```

Comment:
> of all

That sounds weird/ wrong, don't you think?
Maybe we could say that:
```suggestion
        graphene.List(Address, description='The address list of the user.'),
```